gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -161,13 +164,16 @@ public void startUpload(PipedInputStream pipeSource) {
 
   private class UploadOperation implements Callable<Object> {
     // Read end of the pipe.
-    private final BufferedInputStream pipeSource;
+    private final InputStream pipeSource;
```

Comment:
Do we still need `pipeSource` as a field?

Reply:
This was actually removed locally. Forgot to git add before commit ...